docs-ref-services/servicebus.md

```diff
@@ -135,12 +208,14 @@ Create and manage namespaces, topics, queues, and subscriptions with the managem
 
```

Comment:
Please add a note highlighting the fact that the `client` version of the package now has 100% of the management functionality. It's the first package to do this, so it's somewhat surprising IMO.